core/setup.py

```diff
@@ -49,7 +49,7 @@
         # ----
         # dbt-core uses these packages deeply, throughout the codebase, and there have been breaking changes in past patch releases (even though these are major-version-one).
         # Pin to the patch or minor version, and bump in each new minor version of dbt-core.
-        "agate>=1.7.0,<1.8",
+        "agate>=1.7.0,<1.10",
```

Comment:
@mjkanji could you adjust both the lower and upper bounds to match the associated draft PR in `dbt-common` (https://github.com/dbt-labs/dbt-common/pull/119) ?

```python
  "agate>=1.8.0,<1.10",
```

Reply:
Due to some release conflicts we're hitting we're going to keep the lower pin at 1.7 for now.